ADA/databricks_fundamentals.qmd

```diff
@@ -35,9 +37,11 @@ Currently, we are used to using a PC or laptop to do our data processing. A trad
 
 ![](/images/ada-traditional-computer.jpg){width="273"}
```

Comment:
Not related to this PR, but should this have alt text or be aria-hidden? Might be worth an issue to review the images for this?

Reply:
Yeah, every image on the guide should have one or the other but I think approximately none of them do... will raise an issue.

Reply:
Have raised #91 to deal with this - likely to be a big / tricky piece of work but is very important